docs/index.yaml

```diff
@@ -86,34 +86,4 @@ entries:
     urls:
     - https://splunk.github.io/splunk-operator/splunk-operator-2.3.0.tgz
     version: 2.3.0
-  - apiVersion: v2
-    appVersion: 2.2.1
-    created: "2023-07-25T15:50:32.772364-07:00"
-    description: A Helm chart for the Splunk Operator for Kubernetes
-    digest: 8868b9ae2ebde0c667b13c97d71d904a31b5a9f2c803b199bc77324f1727e1fd
-    name: splunk-operator
-    type: application
-    urls:
-    - https://splunk.github.io/splunk-operator/splunk-operator-2.2.1.tgz
-    version: 2.2.1
-  - apiVersion: v2
-    appVersion: 2.2.0
-    created: "2023-07-25T15:50:32.759549-07:00"
-    description: A Helm chart for the Splunk Operator for Kubernetes
-    digest: 49c72276bd7ff93465b0545d8b0814f684cade7d2cd191b6d73d4c3660bd1fb4
-    name: splunk-operator
-    type: application
-    urls:
-    - https://splunk.github.io/splunk-operator/splunk-operator-2.2.0.tgz
-    version: 2.2.0
-  - apiVersion: v2
-    appVersion: 2.1.0
-    created: "2023-07-25T15:50:32.747409-07:00"
-    description: A Helm chart for the Splunk Operator for Kubernetes
-    digest: 34e5463f8f5442655d05cb616b50391b738a0827b30d8440b4c7fce99a291d9a
-    name: splunk-operator
-    type: application
-    urls:
-    - https://splunk.github.io/splunk-operator/splunk-operator-1.0.0.tgz
-    version: 1.0.0
-generated: "2023-07-25T15:50:32.661115-07:00"
```

Comment:
I hope these changes are correct i.e. in line with whatever we have on latest develop/main?

Reply:
yes the only thing done here is to regenerate these files.